src/agenteval/cli.py

```diff
@@ -533,6 +542,11 @@ def view_command(repo_id, config, split, tag, dump_plots, plot_dir):
     is_flag=True,
     help="Ignore git reproducibility checks (not recommended).",
 )
+@click.option(
+    "--dry-run",
```

Comment:
nit:  I usually think of a a dry-run as a read-only operation; maybe `--no-eval` or `--configure-only`